zMax: 2.5
xMax: 2.5
yOffset : 0.05
resolution: 0.05
Scalar: 1.0
Normalizer: 2.5
rate: 1
queue_size: 1
